src/EFCore.PG/Migrations/NpgsqlMigrationsSqlGenerator.cs

```diff
@@ -314,6 +312,47 @@ protected override void Generate(AlterColumnOperation operation, IModel model, M
 
             var type = operation.ColumnType ?? GetColumnType(operation.Schema, operation.Table, operation.Name, operation, model);
 
+            if (operation.ComputedColumnSql != null)
+            {
+                var property = FindProperty(model, operation.Schema, operation.Table, operation.Name);
+
+                // TODO: The following will fail if the column being altered is part of an index.
+                // SqlServer recreates indexes, but wait to see if PostgreSQL will introduce a proper ALTER TABLE ALTER COLUMN
+                // that allows us to do this cleanly.
+                var dropColumnOperation = new DropColumnOperation
+                {
+                    Schema = operation.Schema,
+                    Table = operation.Table,
+                    Name = operation.Name
+                };
+                if (property != null)
+                {
+                    dropColumnOperation.AddAnnotations(_migrationsAnnotations.ForRemove(property));
+                }
+                Generate(dropColumnOperation, model, builder);
```

Comment:
There are no need in brackets here, but an empty line before `Generate`.

Reply:
I think I'm getting influenced by the EF upstream, thanks for reminding me.